### PR TITLE
add extra requirements

### PR DESCRIPTION
**Pull request by gsheni**
- Add extra requirements for featuretools plugins

### CODE REVIEW

**Comment by rwedge** [comment]
Should we test installing these extras as part of our CI tests?

**Comment by kmax12** [comment]
> Should we test installing these extras as part of our CI tests?

might as well

**Comment by gsheni** [comment]
@rwedge PR now tests importing the extra packages.

File: .circleci/config.yml [hunk 1]
```diff
@@ -18,6 +18,18 @@ commands:
...
+    description: "run unit tests on packaged testing files"
+    steps:
+      - run: |
+          virtualenv test_ft_complete_env
+          source test_ft_complete_env/bin/activate
+          python setup.py sdist
+          FEATURETOOLS_VERSION=$(python setup.py --version)
+          tar -zxvf "dist/featuretools-${FEATURETOOLS_VERSION}.tar.gz"
+          pip install -e "featuretools-${FEATURETOOLS_VERSION}/[complete]"
+          python -c "import featuretools_tsfresh_primitives"
```
Comment (rwedge) [inline .circleci/config.yml]: Let's make a second `-run` step in the command that tests the extra imports.  That way the imports don't share the same output log with the pip installation.

Comment (rwedge) [inline .circleci/config.yml]: Can we also test importing `featuretools.tsfresh`

Comment (gsheni) [reply .circleci/config.yml]: Fixed

Comment (gsheni) [reply .circleci/config.yml]: Fixed.

**Pull request merged**